app/models/distribution.rb

```diff
@@ -40,6 +41,8 @@ class Distribution < ApplicationRecord
 
   enum state: { started: 0, scheduled: 5, complete: 10 }
 
+  enum delivery_method: { pick_up: 1, delivery: 2 }
```

Comment:
Based on the change suggestion on using :string, could we use validates presence and inclusion?

Reply:
done